website/docs/r/project_service_identity.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Cloud Platform"
+layout: "google"
+page_title: "Google: google_project_service_identity"
+sidebar_current: "docs-google-project-service-identity"
+description: |-
+ Generate service identity for a service.
+---
+
+# google\_project\_service\_identity
+
+~> **Warning:** These resources are in beta, and should be used with the terraform-provider-google-beta provider.
+See [Provider Versions](https://terraform.io/docs/providers/google/guides/provider_versions.html) for more details on beta resources.
+
+Generate service identity for a service.
+
+~> **Note**: Once created, this resource cannot be updated or destroyed. These
+actions are a no-op.
+
+To get more information about Service Identity, see:
+
+* [API documentation](https://cloud.google.com/service-usage/docs/reference/rest/v1beta1/services/generateServiceIdentity)
+
+<div class = "oics-button" style="float: right; margin: 0 0 -15px">
```

Comment:
You can remove the HTML for Cloud Shell here- Magic Modules handles creating the git repo that backs the link, and since this is handwritten that won't get created.

Reply:
Done.